include/pybind11/numpy.h

```diff
@@ -507,11 +507,16 @@ class dtype : public object {
         return detail::array_descriptor_proxy(m_ptr)->names != nullptr;
     }
 
-    /// Single-character type code.
+    /// Single-character for dtype's kind (ex: float and double are 'f' or int and long int are 'i')
```

Comment:
"float and double" could be "floating point types", and "int and long" could be "integral types". But this works as well.

Reply:
okay !